Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -450,31 +450,29 @@ private static double GetAspectRatio(ScheduleDirect.ImageData i)
 
         private async Task<List<ScheduleDirect.ShowImages>> GetImageForPrograms(
             ListingsProviderInfo info,
-            List<string> programIds,
+            IReadOnlyList<string> programIds,
             CancellationToken cancellationToken)
         {
             if (programIds.Count == 0)
             {
                 return new List<ScheduleDirect.ShowImages>();
             }
 
-            var imageIdString = "[";
-
-            foreach (var i in programIds)
+            StringBuilder str = new StringBuilder("[", 1 + (programIds.Count * 13));
```

Comment:
I know the meaning of 13 becomes apparent when you read the foreach block, but I still think a comment would be nice.